runtime/ops/process.rs

```diff
@@ -530,7 +531,7 @@ fn compute_run_cmd_and_check_permissions(
 }
 
 struct RunEnv {
-  envs: HashMap<String, String>,
+  envs: HashMap<OsString, OsString>,
```

Comment:
Is this a perf optimization? What's the reason to use `OsString` here?

Reply:
It's to keep them in their original encoding. I think it was a bug I introduced earlier to not use `OsString`